sdks/python/apache_beam/ml/transforms/embeddings/open_ai_test.py

```diff
@@ -0,0 +1,291 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import tempfile
+import unittest
+import uuid
+
+import apache_beam as beam
+from apache_beam.ml.inference.base import RunInference
+from apache_beam.ml.transforms import base
+from apache_beam.ml.transforms.base import MLTransform
+
+try:
+  from sdks.python.apache_beam.ml.transforms.embeddings.open_ai import OpenAITextEmbeddings
+except ImportError:
+  OpenAITextEmbeddings = None
+# pylint: disable=ungrouped-imports
+try:
+  import tensorflow_transform as tft
+  from apache_beam.ml.transforms.tft import ScaleTo01
+except ImportError:
+  tft = None
```

Comment:
A tests_requirements.txt file similar to what we do for other frameworks (like the vertex AI one [here](https://github.com/apache/beam/blob/5c4b42f81c67fefec42e28cc6faae52f21ba8f56/sdks/python/apache_beam/ml/inference/vertex_ai_tests_requirements.txt)) would be a good inclusion for test setup.